src/float.jl

```diff
@@ -133,7 +138,7 @@ function parsefield(io::IO, ::Type{T}, opt::CSV.Options, row, col, state, ifnull
             (!(b == LITTLEN || b == BIGN) || eof(io)) && (reset(io); b = readbyte(io); @goto checknullend)
             b = readbyte(io)
             !(b == LITTLEF || b == BIGF) && (reset(io); b = readbyte(io); @goto checknullend)
-            result = T(Inf)
+            result = Inf
```

Comment:
I think we still want these to be `T(Inf)`, right?

Reply:
Woops, right, these are remnants of debugging tests. I've reverted them.